src/elements/content-sidebar/versions/VersionsItem.js

```diff
@@ -65,16 +66,27 @@ const VersionsItem = ({
         is_download_available,
         permissions = {},
         restored_at: restoredAt,
+        retention = {},
         size,
         trashed_at: trashedAt,
         version_number: versionNumber,
         version_promoted: versionPromoted,
     } = version;
     const { can_delete, can_download, can_preview, can_upload } = permissions;
+    const {
+        disposition_action: versionDispositionAction,
+        disposition_at: versionDispositionTime,
+        applied_at: versionAppliedTime,
+    } = retention;
 
     // Version info helpers
     const versionAction = selectors.getVersionAction(version);
     const versionInteger = versionNumber ? parseInt(versionNumber, 10) : 1;
+    const versionDispositionMessage =
+        versionDispositionAction === VERSION_RETENTION_REMOVE_ACTION
+            ? messages.versionRetentionRemove
+            : messages.versionRetentionDelete;
+    const versionDispositionTimestamp = versionDispositionTime ? new Date(versionDispositionTime).getTime() : null;
```

Comment:
Is there a significance to the `null` value or can we do `versionDispositionTime && new Date(versionDispositionTime).getTime()`?